cypress/fixtures/avatars.json

```diff
@@ -0,0 +1,22 @@
+[
+    {
+        "ownerName": "ubiquitybot[bot]",
```

Comment:
This owner I don't recognize and looks wrong

Reply:
It is a test file. Does it matter?